cypress/integration/OCLSubscriptionModule /oclSubscriptionModule.feature 

```diff
@@ -0,0 +1,20 @@
+Feature: OCL Subscription module
+  Background:
+    Given the user is logged in
+
+  @subscription
+  Scenario: The user should be able to click the button to add a new member
+    Given an organization exists
+      And the user is on the organisation detail page
+    When the user clicks on the add new member button
+    Then the user should be on the add new member dialog box
```

Comment:
Might be better to phrase this:

"Then the user should see the add new member dialog box"